src/corona_hakab_model/consts.py

```diff
@@ -1,18 +1,18 @@
 from functools import lru_cache
 from itertools import count
-from typing import NamedTuple
+from typing import NamedTuple, Dict, List
 
 import numpy as np
-from medical_state import ImmuneState, SusceptibleState, ContagiousState
+from medical_state import ImmuneState, SusceptibleState, ContagiousState, MedicalState
 from medical_state_machine import MedicalStateMachine
 from scipy.stats import rv_discrete
 from state_machine import StochasticState, TerminalState
-from util import dist
+from util import dist, upper_bound
 
 
 class Consts(NamedTuple):
     # simulation parameters
-    population_size = 10_000
+    population_size = 100_000
```

Comment:
This changes the memory usage dramatically.
Is there a reason to change the default to 100k?

Reply:
not really, everyone can feel free to play with this number.
i ran the tests before uploading this on 100k because this allows you to see the results of the normalization changes more precisely.

Reply:
I agree that this should remain 10_000 in our branch, so it will be faster to test yourself and see results.
You'll run 100_000 when you need to check that.